static/js/classes/doc.js

```diff
@@ -7,7 +7,7 @@ class Doc {
     this.annotations = [];
   }
 
-   getAnnotationRanges(){
-    return this.annotations.map(function(annotation){return annotation.getRangeAsArray();});
+  getAnnotationRanges() {
+    return this.annotations.map(function (annotation) { return annotation.getRangeAsArray(); });
```

Comment:
This can be removed I believe

Reply:
I'm just going to leave it here in case we need it later